src/components/DocumentActions.tsx

```
@@ -81,44 +81,28 @@ const DocumentActions: React.FC<DocumentActionsProps> = ({
 		setIsPreviewOpen(true);
 	};
 
-	const handleImagePreview = () => {
+	const handleImagePreview = (doc: Document) => {
```

ESLint

[error] 'doc' is defined but never used.

(@typescript-eslint/no-unused-vars)